File: docs/downloads/code/Fibonacci-Number-in-Different-Programming-Languages/C/fibonacci.c
#include <stdio.h>
#include <stdlib.h>

int fib(int n) {
    if(n <= 1) {
        return n;
    }

    int a = 0;
    int b = 1;

    for(int i = 2; i <= n; i++) {
        int c = a + b;
        a = b;
        b = c;
    }
    return b;
}

int main(int argc, char *argv[]) {
    int n = atoi(argv[1]);
    printf("fib(%d) = %d\n", n, fib(n));
    return 0;
}

File: docs/downloads/code/Fibonacci-Number-in-Different-Programming-Languages/JavaScript/fibonacci.js
function fib(n) {
    if (n <= 1) {
        return n
    }

    var a = 0
    var b = 1

    for (let i = 2; i <= n; i++) {
        c = a + b
        a = b
        b = c
    }
    return b
}

const n = process.argv[2]
console.log(`fib(${n}) = ${fib(n)}\n`)
